cytosnake/common/errors.py

```diff
@@ -72,3 +79,75 @@ class WorkflowNotFoundError(BaseFileNotFound):
 
 class WorkflowFailedException(BaseWorkflowException):
     """Raised if a workflow fails during runtime"""
+
+
+class ProjectExistsError(BaseFileExistsError):
```

Comment:
`--force` This is a good point to consider. Unfortunately, `--force` is a flag for `Snakemake`'s API. For example, if you interupt the workflow process, a `lock` is added into the workflow file. Therefore, if you try to run it again, it will will raise an error. `--force` allows the user to bypass that lock that `Snakemake` annoyingly adds.